plugin/github.com.hashicorp.go.kms.wrapping.plugin.v2.proto

```diff
@@ -82,5 +82,21 @@ message HmacKeyIdResponse {
 
 message KeyBytesRequest {};
 message KeyBytesResponse {
-	bytes key_bytes = 10 [json_name="key_bytes"];
+	bytes key_bytes = 10;
```

Comment:
I don't know if we use JSON with these types, but this is technically a breaking change for anyone using the default protobuf JSON marshaler, since it changes the format of the key.

Reply:
adding it back. ty!